<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #323232;
    color: #FFFFFF;
}

QTableWidget {
    background-color: #1E1E1E;
    color: #FFFFFF;
    border: 1px solid #555555;
}

QHeaderView::section {
    background-color: #333333;
    color: #FFFFFF;
}

QPushButton {
    background-color: #1E90FF; /* Blue */
    color: #FFFFFF;
    border: 1px solid #1E90FF;
    padding: 5px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #007FFF; /* Darker Blue on Hover */
    border: 1px solid #007FFF;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    background-color: #f0f0f0;
}

QProgressBar::chunk {
    background-color: rgb(30, 144, 255):
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>39</number>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
